doc/instance.md

```diff
@@ -33,6 +34,7 @@ The following arguments are supported:
 - [public_key](#public_key) - (Optional) Public key to attach to the instance. Mutually exclusive with ssh_key_name.
 - [root_volume_size_in_gb](#root_volume_size_in_gb) - (Optional) Size of the root volume of the instance. This only works for templates that allows root volume resize.
 - [private_ip](#private_ip) - (Optional) Instance's private IPv4 address.
+- [dedicated_group_id](#dedicated_group_id) - (Optional) Dedicated group id in which the instance will be created
```

Comment:
Should we say dedicated group ID or affinity group ID. Will it be clear to the user if the resource type is *affinity group*?